src/coreclr/jit/lsraloongarch64.cpp

```diff
@@ -723,7 +723,7 @@ int LinearScan::BuildCall(GenTreeCall* call)
         {
             // Fast tail call - make sure that call target is always computed in volatile registers
             // that will not be overridden by epilog sequence.
-            ctrlExprCandidates = allRegs(TYP_INT) & RBM_INT_CALLEE_TRASH;
+            ctrlExprCandidates = (allRegs(TYP_INT) & RBM_INT_CALLEE_TRASH) ^ RBM_GSCOOKIE_TMP;
```

Comment:
```suggestion
            ctrlExprCandidates = (allRegs(TYP_INT) & (RBM_INT_CALLEE_TRASH & ~RBM_GSCOOKIE_TMP));
```

I don't see good reason to add an unnecessary assumption that the LHS contains these registers already.

Reply:
As there are no suitable RBM mask, so I add a additional `RBM_GSCOOKIE_TMP`.